CHANGELOG.md

```diff
@@ -1,15 +1,111 @@
+  1. Enforce `jsx-space-before-closing` (#3397)
+  1. Enforce `id-blacklist` and blacklist `e` (#3398)
+
+##### Component Fixes / Enhancements
+- [SvgIcon] Use stateless functional component instead of `React.createClass` (#3326)
+- [DropDownMenu] Fix `openImmediately` regression (#3384)
+- [DropDownMenu] Expose Menu listStyle property (#3294)
+- [DropDownMenu] Safari select-field fix (#3175)
+- [Menu] Safari select-field fix (#3175)
+- [Menu] Fix _isChildSelected child not recognising first child (#3165)
+- [Menu] Fix a regressionthat would apply the select style to all the MenuItems (#3244)
+- [IconMenu] Document `multiple` property of Menu (#3223)
+- [IconMenu] Support MenuItem nested menuItems (#3265)
+- [IconMenu] enable `useLayerForClickAway` (#3400)
+- [IconButton] Fixed tooltip ripple size for IE (#3016)
+- [LeftNav] Add iOS momentum scroll (#2946)
+- [RaisedButton] Fix alignment between text and icons (#3366)
+- [RaisedButton] Customizable ripple effect style (#3368)
+- [FlatBotton] Fix alignment between text and icons (#3380)
+- [FloatingActionButton] Expose Paper zDepth (#3387)
+- [List] Fix issue with styling on list related components (#3278)
+- [ListItem] Fix hardcoded `secondaryTextColor` (#3288)
+- [Table] Send event object after click, hover, hoverOut on cell (#3002)
+- [TextField] Add textareaStyle property (#3238)
+- [TextField] Fix wrong label id (#3240)
+- [TextField] Fix defaultValue behavior (#3239)
+- [TextField] Fixed a bug where clicking on floating label and typing simultaneuosly loses keypress (#3055)
+- [TextField] Fixed ie9-ie10 click focus problem (#3193)
+- [TimePicker] Update time state on new defaultTime prop (#3095)
+- [TouchRipple] Abort on scroll (#3407)
+- [Toggle] Fixes styling issue (#3299)
+- [ToolbarTitle] Fix overflow (#3250)
+- [Card] support for controlled expansion (#3258) (Thanks to @cgestes)
+- [CardActions] Allow to accept false as child (#3215)
+- [Checkbox] Disabled style error fix (#3432)
+- [AutoComplete] Apply various improvement (#3214) (Thanks to @oliviertassinari)
+- [AutoComplete] Fix the focus / blur issue (#3356) (Thanks to @oliviertassinari)
+- [AutoComplete] Apply the style property only on the root component (#3243)
+- [AutoComplete] Added `maxSearchResults` property (#3262)
+- [Autocomplete] Disable browser default autocomplete popup (#3253)
+- [InkBar] remove `&nbsp;` (#3283)
+- [EnhancedTextArea] Provide various style fixes (#3277)
+- [EnhancedButton] fix enhanced buttons containing a link instead of a button (#3303)
+- [EnhancedSwitch] Added inputStyle prop to enhanced switch (#1693)
+- [RadioButton] Allow customising icons (#3285)
+- [Slider] Remove style-propable mixin and react-dom (#3332) (Thanks to @felipethome)
+
+##### Deprecations
+- [Menu] Deprecated built in `animated` (#3216)
+- [Core] Deprecated `style-propable` mixin (#3351)
+- [Core] Deprecated `ThemeDecorator` in favor of `MuiThemeProvider` (#3267)
+- [Core] Deprecated `theme-manager` and `raw-themes` (#3267)
```

Comment:
The export for `raw-themes` was deprecated, but I wonder if we should make a special note at the top of the release notes that accessing the rawTheme in muiTheme is deprecated but will not produce a warning. It'd be good to kill that alias in `0.16.0`.